public/fonts/fontello.html

```diff
@@ -229,48 +229,48 @@
 }
 @font-face {
       font-family: 'fontello';
-      src: url('./fontello.eot?81635713');
-      src: url('./fontello.eot?81635713#iefix') format('embedded-opentype'),
-           url('./fontello.woff?81635713') format('woff'),
-           url('./fontello.ttf?81635713') format('truetype'),
-           url('./fontello.svg?81635713#fontello') format('svg');
+      src: url('./font/fontello.eot?45674533');
+      src: url('./font/fontello.eot?45674533#iefix') format('embedded-opentype'),
+           url('./font/fontello.woff?45674533') format('woff'),
+           url('./font/fontello.ttf?45674533') format('truetype'),
+           url('./font/fontello.svg?45674533#fontello') format('svg');
```

Comment:
You are right, i didn't check that file I simply put it inside t he folder. But is it actually needed for the project? I feel it is just to check on the output.